docs/source/staking/stake_management.rst

```diff
@@ -422,15 +422,15 @@ Collect Staker Rewards
 ~~~~~~~~~~~~~~~~~~~~~~
 
 NuCypher nodes earn two types of rewards: staking rewards (in NU) and policy fees (i.e., service fees in ETH).
-To collect these rewards use ``nucypher stake collect-reward`` with flags ``--staking-reward`` and ``--policy-fee``
+To collect these rewards use ``nucypher stake rewards withdraw`` with flags ``--staking-reward`` and ``--policy-fee``
```

Comment:
What do you think about simplifying the `--policy-fee` flag to merely `--fees` and `--staking-reward` to `--tokens`? If not, the original flags should be plural, I just realized that.

Reply:
Sounds good. We should also rename negative switches, so `--no-tokens` and `--no-fees`.